src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -669,5 +670,398 @@ private boolean validateFieldValue(String fieldKey, String fieldValue) {
+                    StringBuffer str = new StringBuffer();
+                    List<String> errList = new ArrayList<>();
+                    Map<String, Object> valuesToBeUpdate = new HashMap<>(record.toMap());
+                    Map<String, Object> userDetails = null;
+                    boolean isEmailOrPhoneNumberValid = false;
+
+                    // Validate email and phone presence
+                    String email = record.size() > 1 ? record.get(1) : null;
+                    String phone = record.size() > 2 ? record.get(2) : null;
+                    boolean emailExists = email != null && !email.isEmpty();
+                    boolean phoneExists = phone != null && !phone.isEmpty();
+                    if (!emailExists && !phoneExists) {
+                        errList.add("Email or Phone is missing");
+                    }
+
+                    // Validate email
+                    if (emailExists) {
+                        if (ValidationUtil.validateEmailPattern(email)) {
+                            userDetails = new HashMap<>();
+                            isEmailOrPhoneNumberValid = this.verifyUserRecordExists("EMAIL", email, userDetails);
+                        } else {
+                            errList.add("Invalid Email format");
+                        }
+                    }
+
+                    // Validate phone
+                    if (!isEmailOrPhoneNumberValid) {
+                        if (ValidationUtil.validateContactPattern(phone)) {
+                            userDetails = new HashMap<>();
+                            isEmailOrPhoneNumberValid = this.verifyUserRecordExists("PHONE", phone, userDetails);
+                        } else {
+                            errList.add("Invalid Phone number format");
+                        }
+                    }
+
+                    if (!isEmailOrPhoneNumberValid) {
+                        errList.add("User record does not exist with given email and/or Mobile Number");
+                    } else {
+                        errList.clear();
+                        String userRootOrgId = (String) userDetails.get(Constants.ROOT_ORG_ID);
+                        String mdoAdminRootOrgId = inputDataMap.get(Constants.ROOT_ORG_ID);
+                        if (!mdoAdminRootOrgId.equalsIgnoreCase(userRootOrgId)) {
+                            errList.add("The User belongs to a different MDO Organisation");
+                            valuesToBeUpdate.put("Error Details", String.join(",", errList));
+                            failedRecordsCount++;
+                            totalRecordsCount++;
+                            continue;
+                        }
+                    }
+
+                    // Organization-specific validations
+                    if (!isEmailOrPhoneNumberValid) {
+                        errList.add("User record does not exist or is invalid");
+                    } else {
```

Comment:
this is already checked at line 742 -- check only once... so that we have only one error message